src/libs/actions/BankAccounts.js

```diff
@@ -816,6 +816,27 @@ function updateReimbursementAccountDraft(bankAccountData) {
     Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT, bankAccountData);
 }
 
+/**
+ * Checks the given number is a valid US Routing Number
+ * @param {String} number
+ * @returns {Boolean}
+ */
+function validateRoutingNumber(number) {
+    let n = 0;
+    for (let i = 0; i < number.length; i += 3) {
+        n += (parseInt(number.charAt(i), 10) * 3)
+        + (parseInt(number.charAt(i + 1), 10) * 7)
```

Comment:
I believe you don't need these parens
```suggestion
        n += parseInt(number.charAt(i), 10) * 3
        + parseInt(number.charAt(i + 1), 10) * 7
```

Reply:
There is some lint warning so add
<img width="587" alt="Screenshot 2021-10-20 at 8 03 48 PM" src="https://user-images.githubusercontent.com/85645967/138113818-ade446c7-36ee-4e06-bdd1-782ac97c0d5e.png">
ed